templates/rhel8.tpl.yaml

```diff
@@ -45,7 +45,7 @@ metadata:
 {% if item.default %}
     template.kubevirt.io/default-os-variant: "true"
 {% endif %}
-    template.kubevirt.io/architecture: "{{ 'amd64' if target_arch == 'x86_64' else target_arch }}"
+    template.kubevirt.io/architecture: "{{ 'amd64' if target_arch == 'x86_64' else 'arm64' if target_arch == 'aarch64' else target_arch }}"
```

Comment:
There are no common-instancetypes for rhel8 on ARM. I'd keep that out for now.

Reply:
You can drop changes to `rhel8.tpl.yaml`

Reply:
Right! Thank you!